pom.xml

```diff
@@ -197,5 +197,10 @@ Jean-Yves Tinevez and Michael Zinsmaier.</license.copyrightOwners>
 			<artifactId>junit-benchmarks</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>sc.fiji</groupId>
+			<artifactId>bigdataviewer-vistools</artifactId>
```

Comment:
The Maven build on Travis now fails because `bigdataviewer-vistools` contains a transient dependency on `imglib2-roi`, creating a circular dependency.

As this is a `test` scope dependency only, the issue can be avoided by adding an exclusion:

```
		<dependency>
			<groupId>sc.fiji</groupId>
			<artifactId>bigdataviewer-vistools</artifactId>
			<scope>test</scope>
			<exclusions>
				<exclusion>
					<groupId>net.imglib2</groupId>
					<artifactId>imglib2-roi</artifactId>
				</exclusion>
			</exclusions>
		</dependency>
```

I'm not sure though if it is the best way to address this.

Reply:
Interesting hack—I didn't know that worked.Personally, I'd vote for the usage examples requiring vistools to move downstream. Perhaps into vistools itself, since it already depends on imglib2-roi transitively.